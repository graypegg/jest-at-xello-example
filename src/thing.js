/**
 * Adds the first two parameters together and returns the result.
 * @param {number} firstNumber 
 * @param {number} secondNumber 
 * @returns {number}
 */
export function sum (
  firstNumber,
  secondNumber
) {
  const element = document.createElement('div')
  element.appendChild(
    document.createTextNode(
      firstNumber + secondNumber
    )
  )
  return element
}