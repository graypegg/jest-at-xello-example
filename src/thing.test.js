import * as thing from './thing.js';

describe('Thing', () => {
  it('should export sum method', () => {
    expect(thing.sum).toBeTruthy()
  })

  describe('Thing.sum', () => {
    it('should take together two numbers and return their sum', () => {
      expect(thing.sum(2, 5)).toBe(7)
    })
  })
})