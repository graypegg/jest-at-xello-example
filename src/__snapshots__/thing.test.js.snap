// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Thing Thing.sum should take together two numbers and return their sum 1`] = `
<div>
  7
</div>
`;
